app-modules/service-management/src/Models/ServiceRequest.php

```diff
@@ -145,6 +146,13 @@ public function createdBy(): BelongsTo
         return $this->belongsTo(User::class);
     }
 
+    // TODO We need a way to correspond custom statuses to some normalized data set
+    // So that we can determine what an "open" status is in the context of a client's data
```

Comment:
This will be very important before we have a folks using this I think. Do you know if we have a ticket for this already?

Reply:
I do not think so - I'll add one to the backlog.